lib/model/image.py

```diff
@@ -7,7 +7,27 @@
 from pdqhashing.hasher.pdq_hasher import PDQHasher
 from lib import schemas
 
+from torchvision import transforms
+import torch
+from lib.logger import logger
+import numpy as np
+from PIL import Image
+
+IMAGENET_NORMALIZATION_MEAN = [0.485, 0.456, 0.406]
+IMAGENET_NORMALIZATION_STD = [0.229, 0.224, 0.225]
 class Model(Model):
+    def __init__(self):
+        super().__init__()
+        #FIXME: Load from a Meedan S3 bucket
+        try:
+            self.model = torch.jit.load("sscd_disc_mixup.torchscript.pt")
```

Comment:
Instead of the generic self.model, we should redact to something clearer like self.sscd_model